### DISCUSSION TITLE
Rule proposal: `template/self-closing-tags`

### DISCUSSION THREAD

**Discussion by aleesaan** [angular-eslint/angular-eslint#1318] [New Rule Proposals]
[Angular 15.1.0](https://github.com/angular/angular/releases/tag/15.1.0) has introduced [self-closing tags for custom elements](https://github.com/angular/angular/pull/48535), can we get a rule for that? It would be great if it supported autofix too 😇

**Comment by json-derulo** [comment]
This rule was introduced with v16.1.0 and it also includes an autofix, so I think this discussion can be marked as resolved.

**Comment by jonaaix** [comment]
@json-derulo  Where can I find the documentation?

**Reply by aleesaan** [reply]
Here it is!  
https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/docs/rules/prefer-self-closing-tags.md

**Reply by jonaaix** [reply]
@aleesaan thanks! also got it with try-error 😅 

```json
  {
     "files": ["*.html"],
     "extends": ["plugin:@angular-eslint/template/recommended"],
     "rules": {
        "@angular-eslint/template/prefer-self-closing-tags": "error"
     }
  }
```
